### DISCUSSION TITLE
Is it possible to save spacing on cross screen jumps with random mod?

### DISCUSSION THREAD

**Discussion by Zoleks** [ppy/osu#15465] [Q&A] [answered]
Random mode totally cuts starrate on maps with cross screen jumps like Monstrata's Quaver [Crescendo] difficulty how example, it have 7.28* without any mods and ~6.5* with random mod. One more example - loved Kigurumi Sungeki have 7.62* nomod and ~6.7* with random mod, almost 1* is cutting.

**Comment by Loreos7** [comment]
Maybe this would be helpful:

![12312333158](https://user-images.githubusercontent.com/86934170/142931930-171bca68-eaa7-4889-bd89-eb6f3ec7a196.jpg)

**Reply by bdach** [reply]
you have no idea how complicated making this switch button work properly would be.

**Reply by Loreos7** [reply]
I know that, actually. Maybe someone will take it up, for some reason.

**Comment by smoogipoo** [comment, marked as answer]
I'm going to say no to this. By definition random mod randomises the map. To preserve star rating you have to preserve the map, you basically can only rotate it and even then it's not guaranteed to preserve star rating (or may not in the future).
